common/test/acceptance/tests/studio/test_studio_html_editor.py

```diff
@@ -365,3 +366,29 @@ def test_font_selection_dropdown(self):
         }
         self.html_editor.open_font_dropdown()
         self.assertDictContainsSubset(EXPECTED_FONTS, self.html_editor.font_dict())
+
+    def test_link_plugin_adds_foreign_language_character_properly(self):
```

Comment:
Can't we handle this special case in the `JS` tests? Not thrilled on the choice of bokchoy test for this case.

Reply:
sure.

Reply:
@DawoudSheraz  I just saw the tests and we are faking the external visual TinyMCE editor and as the change, we have made is the library, I can't find an easy way to test it in Jasmine. 

https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/js/spec/html/edit_spec.js#L8

Reply:
Ok. We can let the bokchoy test remain in its place.